README.md

```diff
@@ -6,4 +6,40 @@ Travis:   [![Travis](https://travis-ci.org/aspnet/MvcPrecompilation.svg?branch=d
 
 The Razor syntax provides a fast, terse, clean, and lightweight way to combine server code with HTML to create dynamic web content. This repo contains tooling that allows compilation of MVC Razor views as part of build and publish.
 
+## Installation and usage
+Add a package reference to `Microsoft.AspNetCore.Mvc.Razor.ViewCompilation` in your project:
```

Comment:
Not on desktop. I guess I can say if you aren't referencing the .All package, use this.